easy_rec/python/model/easy_rec_model.py

```diff
@@ -47,7 +48,7 @@ def __init__(self,
     self._emb_reg = regularizers.l2_regularizer(self.embedding_regularization)
     self._l2_reg = regularizers.l2_regularizer(self.l2_regularization)
     # only used by model with wide feature groups, e.g. WideAndDeep
-    self._wide_output_dim = -1
+    self._wide_output_dim = self.get_wide_output_dim()
```

Comment:
it is only used by wide_and_deep, may be put in WideAndDeep model only, should not be placed in base model